### PR TITLE
Add initial notes from AM6 install

### PR DESCRIPTION
**Pull request by pnorman**

### CODE REVIEW

File: hosting/sites/am6/inventory.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- 1 broken 3.5" HP disk tray
+- 1 HP flash backed write cache, PN 571436-002
+- 1 HP raid card PN 575227-B21 with FBWC PN 570502-002
+- 4 HP fans
+
+### On top of boxes
+- 2 750W HP supply in tray
+- Fujitsu CA06731-B100
+- 1 HP 146GB 10K SAS drive in tray
+- DB9 to 6 wire phone connector
```
Comment (Firefishy) [inline hosting/sites/am6/inventory.md]: APC PDU Serial (DB9 to RJ12)

File: hosting/sites/am6/inventory.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- Spare labels for unused switch ports
+- Rack nuts, rack screws, and washers. Over 50
+- Black duct tape
+- Black velcro cable ties
+
+### Box 4
+- 12 disk trays made from stamped steel
+- 11 3.5" HP disk trays
+- 1 broken 3.5" HP disk tray
+- 1 HP flash backed write cache, PN 571436-002
```
Comment (Firefishy) [inline hosting/sites/am6/inventory.md]: (for P410)

File: hosting/sites/am6/inventory.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- 1 HP raid card PN 575227-B21 with FBWC PN 570502-002
```
Comment (Firefishy) [inline hosting/sites/am6/inventory.md]: HP Smart Array P411 Controller for MSA60. (Cabled Needed)

**Comment by Firefishy** [comment]
I feel a little nervous giving so much detail publicly on how access works to the datacenter. Private OPS wiki?

**Comment by pnorman** [comment]
We could move it private, but there's not that much info on access really. The address is public. Google turns up most of the Equinix PDFs we got.

We should also review the shipping procedures for the same issues.

File: hosting/sites/am6/inventory.md [hunk 4]
```diff
@@ -35,12 +35,12 @@ IEC power cables
...
+- 1 HP smart array P411 controller for MSA60 PN 575227-B21 with FBWC PN 570502-002
 - 4 HP fans
 
 ### On top of boxes
 - 2 750W HP supply in tray
 - Fujitsu CA06731-B100
 - 1 HP 146GB 10K SAS drive in tray
-- DB9 to 6 wire phone connector
\ No newline at end of file
+- APX PSU Serial DB9 to RJ12
```
Comment (Firefishy) [inline hosting/sites/am6/inventory.md]: APC ;-)

**Comment by pnorman** [comment]
I can move it there. I wrote this on the train, so plain Markdown files were the only option.

**Pull request closed without merging**